wafer/sponsors/templates/wafer.sponsors/sponsors_footer.html

```diff
@@ -1,21 +1,23 @@
 {% load sponsors %}
 <div id="wafer-sponsors" class="container">
-  {% for package in packages %}
-    {% if package.sponsors.exists %}
-      {% for sponsor in package.sponsors.all %}
-        {% sponsor_tagged_image sponsor "footer_logo" as logo_url %}
-        <a href="{{ sponsor.url }}">
-          {% if logo_url != '' %}
-            <img src="{{ logo_url }}"
-                 alt="{{ sponsor.name }}"
-                 style="max-width: 200px; max-height:100px;"
-                 title="{{ sponsor.name }}">
-          {% else %}
-            <span class="nologo">{{ sponsor.name }}</span>
-          {% endif %}
-        </a>
-      {% endfor %}
-      <br>
-    {% endif %}
-  {% endfor %}
+  {% spaceless %}
```

Comment:
Possibly the spaceless block only needs to be around the `<a ...>...</a>` tag? I think have some whitespace around the rest makes some sort of semantic sense (i.e. its white space between sponsors and around things like the br tag).